### ISSUE TITLE
Add mode for plotting the complex plane

### ISSUE THREAD

**Issue by matthew4850** [microsoft/calculator#945]
**Problem Statement**
Allow for the plotting of complex numbers and loci in an argand diagram

**Evidence or User Insights**
This would be greatly beneficial to students learning about complex numbers. This is part of the further maths syllabus for high school students in the UK, I am not sure about other countries.

**Goals**
Allow for plotting of complex numbers and loci in the form
- x+iy
- re^iθ
- |z|=r
- |z-(a+bi)|=|z-(x+iy)|

**Non-Goals**
Non-goals include things such as visualisation of transformations of the complex plane.

**Requested Assignment**
I'm just suggesting this idea.  I don't want to implement it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
Thanks for the great suggestion! This is likely dependent on #613.

**Comment by grochocki** [comment]
@matthew4850 Thanks for your submission! We think this is potentially very interesting, and while plotting complex numbers certain is a topic in mathematics, it is unclear how widespread this is in curriculums. We would like to keep this issue open to collect more data to better understand how common working with these types of equations is in and out for the classroom before moving forward with it. If you can provide additional support/justification, that would really help. Otherwise, we will keep it open to community discussion.
